garak/generators/base.py

```diff
@@ -47,11 +52,26 @@ def __init__(self, name="", generations=10):
                 self.fullname = self.name
         if not self.generator_family_name:
             self.generator_family_name = "<empty>"
+        if hasattr(self, "ENV_VAR"):
+            if not hasattr(self, "key_env_var"):
+                self.key_env_var = self.ENV_VAR
+        self._validate_evn_var()
+
         print(
             f"🦜 loading {Style.BRIGHT}{Fore.LIGHTMAGENTA_EX}generator{Style.RESET_ALL}: {self.generator_family_name}: {self.name}"
         )
         logging.info("generator init: %s", self)
 
+    def _validate_evn_var(self):
+        if hasattr(self, "key_env_var"):
+            if not hasattr(self, "api_key") or self.api_key is None:
+                self.api_key = os.getenv(self.key_env_var, default=None)
+                if self.api_key is None:
+                    raise ValueError(
+                        f'Put the {self.generator_family_name} API key in the {self.key_env_var} environment variable (this was empty)\n \
```

Comment:
Let's really get their attention to this line
```suggestion
                        f'🛑 Put the {self.generator_family_name} API key in the {self.key_env_var} environment variable (this was empty)\n \
```